lib/wrapbox/runner/ecs.rb

```diff
@@ -36,7 +38,25 @@ def initialize(options)
         @revision = options[:revision]
         @cluster = options[:cluster]
         @region = options[:region]
+
+        if options[:container_definition] && options[:task_definition]
+          raise "Please set only one of `container_definition` and `task_definition`"
+        end
         @container_definition = options[:container_definition]
+        @task_definition_info = options[:task_definition]
+
+        if @container_definition
+          @task_definition_name = "wrapbox_#{@name}"
+          @main_container_name = @container_definition[:name] || @task_definition_name
+        elsif @task_definition_info
+          @task_definition_name = @task_definition_info[:task_definition_name]
+          @main_container_name = @task_definition_info[:main_container_name]
+        end
+
+        unless @main_container_name
```

Comment:
[imo] I prefer to move this code into `elsif @task_definition_info` block because this error implys expects the case where `task_definition` is defined.